packages/yew/src/html/classes.rs

```diff
@@ -65,16 +66,21 @@ impl Classes {
 }
 
 impl IntoPropValue<AttrValue> for Classes {
+    #[inline]
     fn into_prop_value(mut self) -> AttrValue {
         if self.set.len() == 1 {
-            self.set.pop().unwrap()
+            match self.set.pop() {
+                Some(attr) => attr,
+                None => unsafe { unreachable_unchecked() },
```

Comment:
Add a comment here explaining why it this is `unreachable` and explain the safety argument.